presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkConfiguration.java

```diff
@@ -23,10 +23,13 @@
 
 public class PrestoSparkConfiguration
 {
+    public static final String METADATA_STORAGE_TYPE_KEY = "metadata_storage_type";
+    public static final String METADATA_STORAGE_TYPE_LOCAL = "LOCAL";
```

Comment:
Depends on how we plan to have more enumeration of such key-value pairs, having a more centralized place is better than putting them here.

Reply:
The plan is to have the possibility to add more options for internal use. Unfortunately, it cannot be centrally enumerated in the OSS code. Not sure about a cleaner solution. I feel it's still better than having string constants repeated in the code as it was prior to this change.